### DISCUSSION TITLE
Highlighting php code embedded into html file extension

### DISCUSSION THREAD

**Discussion by jfoclpf** [community/community#21318] [Other Feature Feedback, Questions, & Ideas] [answered]
Just to let you know that PHP code is not being correctly highlighted when it is embedded inside HTML code (using html file extension)

```html
<!DOCTYPE html>
<html>
    <head>
    </head>
    <body>
    <?php
       if (count($_GET) == 0) {
         echo "Hello World";
       }
    ?>
    </body>
</html>
```

Example also [here](https://github.com/jfoclpf/in-my-district/blob/main/website/snippets/show-ocurrences-by-authority.html).

It would be nice if you could have this improvement of PHP highlighting everything inside `<?php ?>` when the code extension is html

**Comment by colinwm** [comment, marked as answer]
If you rename your file with the `.html.php` extension, the highlighting will work

**Reply by jfoclpf** [reply]
@colinwm  niiiice

but I have in that HTML file also Javascript code embedded into `<script>` tags

Shall I use `.html.js.php` ?

**Reply by jfoclpf** [reply]
JS was already being highlighted.

Thanks again
